editor/editor_interface.h

```diff
@@ -110,6 +122,14 @@ class EditorInterface : public Object {
 	String get_current_feature_profile() const;
 	void set_current_feature_profile(const String &p_profile_name);
 
+	// Editor dialogs.
+
+	// TODO: currently crashes if used when !is_editor_hint(), but https://github.com/godotengine/godot/pull/80962 should fix
```

Comment:
The PR was merged, so is this still true?

Reply:
You're right, it doesn't crash now! Instead, execution aborts and I get: "Invalid call. Nonexistent function 'popup_dialog_scene_tree' in base 'EditorInterface'." As mentioned in the linnked PR, the error message leaves a lot to be desired, but I'll remove the TODO.